library/validator-assets/src/helixstreet.tsx

```diff
@@ -0,0 +1,28 @@
+const helixstreet = () => (
```

Comment:
Just to follow conventions - upper case file name and component name.

```suggestion
const Helixstreet = () => (
```